server/questions.js

```diff
@@ -90,7 +89,11 @@ var prepareQuestionData = function(question, callback){
             questionToAdd.type = common.questionTypes.CHOOSEALL.value;
             questionToAdd.choices = question.choices;
             questionToAdd.answer = question.answer;
+            break;
 
+        case common.questionTypes.ORDERING.value:
+            questionToAdd.type = common.questionTypes.ORDERING.value;
+            questionToAdd.orderList = question.orderList;
```

Comment:
questionToAdd.answer
Lets keep it consistent, each question has an answer field